WeakAuras/Transmission.lua

```diff
@@ -372,6 +372,7 @@ local function install(data, oldData, patch, mode)
     if patch then -- if there's no result from Diff, then there's nothing to do
       for key in pairs(patch) do
         local checkButton = keyToButton[key]
+        checkButton = (not isParent and checkButton == checkButtons.anchor) and checkButtons.arrangement or checkButton
```

Comment:
I kinda prefer:
if (not isParent and checkButton == checkButtons.anchor) then
  checkButton = checkButtons.arrangement;
end

Even if longer, that makes imho the intent of changing the checked button in one specific case cleare.